grabbit/src/main/groovy/com/twcable/grabbit/server/services/impl/DefaultServerService.groovy

```diff
@@ -41,38 +41,41 @@ import javax.servlet.ServletOutputStream
 @Component(label = "Grabbit Server Service", description = "Grabbit Server Service", immediate = true, metatype = true, enabled = true)
 @Service(ServerService)
 @SuppressWarnings('GroovyUnusedDeclaration')
-class DefaultServerService implements ServerService{
+class DefaultServerService implements ServerService {
 
-    @Reference(bind='setSlingRepository')
+    @Reference(bind = 'setSlingRepository')
     SlingRepository slingRepository
 
-    @Reference(bind='setConfigurableApplicationContext')
+    @Reference(bind = 'setConfigurableApplicationContext')
     ConfigurableApplicationContext configurableApplicationContext
 
 
-    void getContentForRootPath(@Nonnull String path, @Nullable String afterDateString, @Nonnull ServletOutputStream servletOutputStream) {
+    void getContentForRootPath(
+            @Nonnull String path,
+            @Nonnull String excludePaths,
+            @Nullable String afterDateString, @Nonnull ServletOutputStream servletOutputStream) {
 
-        if(path == null) throw new IllegalStateException("path == null")
-        if(servletOutputStream == null) throw new IllegalStateException("servletOutputStream == null")
+        if (path == null) throw new IllegalStateException("path == null")
+        if (excludePaths == null) throw new IllegalStateException("excludePaths == null")
```

Comment:
Why does "excludePaths" have to be required? In this context, if it's missing it could be treated as an empty String/collection.